packages/flutter/test/material/calendar_date_picker_test.dart

```diff
@@ -1185,24 +1185,24 @@ void main() {
         theme: theme,
       ));
 
-      RenderObject inkFeatures = tester.allRenderObjects.firstWhere((RenderObject object) => object.runtimeType.toString() == '_RenderInkFeatures');
```

Comment:
It's nice to see `allRenderObjects` being removed here (and all over the place). I don't know if it will make a noticeable difference when running these tests but the new `Material.of` approach must be a lot more performant.

Reply:
Agreed! I'm not 100% sure how the different `tester` methods compare, but if we see a sizeable performance boost that'd be awesome, and it's fun to make it more readable regardless :)